collectors/freeipmi.plugin/multi_metadata.yaml

```diff
@@ -159,127 +161,32 @@ modules:
         - freeipmi
       most_popular: true
     overview:
```

Comment:
everything from this line and downwards cat be removed

Reply:
do you want me to remove it to merge? It still passes the schema, and the yaml needs work anyways